test/spec/services/taskServicesSpec.js

```diff
@@ -161,8 +161,6 @@ describe('Tasks Service', function() {
     it('toggles the _editing property', function() {
```

Comment:
This is no longer accurate. You made it so that it doesn't toggle it. It just sets it to true.